### PR TITLE
Rentals & RaR - SBA section: Create 'Structures and buildings allowan…

### PR DESCRIPTION
**Pull request by willjshark**
…ce qualifying amount' page

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -661,6 +661,9 @@ POST        /:taxYear/rentals-rent-a-room/allowances/replacement-of-domestic-goo
 GET         /:taxYear/rentals-rent-a-room/allowances/change-replacement-of-domestic-goods               controllers.allowances.ReplacementOfDomesticGoodsController.onPageLoad(taxYear: Int, mode: Mode = CheckMode,  propertyType: PropertyType = RentalsRentARoom)
 POST        /:taxYear/rentals-rent-a-room/allowances/change-replacement-of-domestic-goods               controllers.allowances.ReplacementOfDomesticGoodsController.onSubmit(taxYear: Int, mode: Mode = CheckMode,  propertyType: PropertyType = RentalsRentARoom)
 
+GET         /:taxYear/rentals-rent-a-room/allowances/check-your-answers                                  controllers.rentalsandrentaroom.allowances.RentalsAndRentARoomAllowancesCheckYourAnswersController.onPageLoad(taxYear: Int)
```
Comment (arunkumar461) [inline conf/app.routes:664]: Allowances CYA page part of this ticket?

File: conf/messages.en [hunk 2]
```diff
@@ -1075,8 +1075,8 @@ structureBuildingQualifyingDate.error.maxDate = Qualifying date for structures a
 structureBuildingQualifyingDate.error.minDate = Qualifying date for structures and buildings allowance must be after 29 October 2018
 structureBuildingQualifyingDate.change.hidden = Qualifying date
 
-structureBuildingQualifyingAmount.title = Qualifying amount
-structureBuildingQualifyingAmount.heading = Qualifying amount
+structureBuildingQualifyingAmount.title = Structures and buildings allowance qualifying amount
```
Comment (arunkumar461) [inline conf/messages.en:1078]: Welsh file update pls

**Review by arunkumar461 — CHANGES_REQUESTED** [review]

**Review by vivekhmrc — DISMISSED** [review]

File: conf/app.routes (hunk 1 above)
Comment (willjshark) [reply conf/app.routes:664]: No just putting the route in the correct place because it was in the SBA section

File: conf/messages.en (hunk 2 above)
Comment (willjshark) [reply conf/messages.en:1078]: Done

**Review by arunkumar461 — APPROVED** [review]

**Pull request merged**